cmd/minikube/cmd/start.go

```diff
@@ -338,12 +340,41 @@ func runStart(cmd *cobra.Command, args []string) {
 		ssh.SetDefaultClient(ssh.External)
 	}
 
-	kubeconfig, err := startNode(existing, mc, n)
-	if err != nil {
-		exit.WithError("Starting node", err)
+	var existingAddons map[string]bool
+	if viper.GetBool(installAddons) {
+		existingAddons = map[string]bool{}
+		if existing != nil && existing.Addons != nil {
+			existingAddons = existing.Addons
+		}
 	}
 
-	if err := showKubectlInfo(kubeconfig, k8sVersion, mc.Name); err != nil {
+	kubeconfig := node.Start(cc, n, existingAddons, true)
+
+	numNodes := viper.GetInt(nodes)
+	if numNodes == 1 && existing != nil {
+		numNodes = len(existing.Nodes)
+	}
+	if numNodes > 1 {
+		if driver.BareMetal(driverName) {
+			out.T(out.Meh, "The none driver is not compatible with multi-node clusters.")
+		} else {
+			for i := 1; i < numNodes; i++ {
+				nodeName := fmt.Sprintf("m%02d", i+1)
```

Comment:
I noticed this formatting in two locations. Add a `node.Name(int)` function?

Reply:
yep, added.